activityMonitor.js

```diff
@@ -0,0 +1,54 @@
+import os from "os"
+import fs from "fs"
+import childProcess from "child_process"
+
+const WINDOWS_COMMAND = `powershell "Get-Process | Sort-Object CPU -Descending | Select-Object -Property Name, CPU, WorkingSet -First 1 | ForEach-Object { $_.Name + ' ' + $_.CPU + ' ' + $_.WorkingSet }"`
+const UNIX_LIKE_OS = "ps -A -o %cpu,%mem,comm | sort -nr | head -n 1"
+
+const isWindows = () => {
+  return os.platform() === "win32"
+}
+
+const getCommands = () => {
+  return isWindows() ? WINDOWS_COMMAND : UNIX_LIKE_OS
+}
+
+const execProcess = (command, cb) => {
+  childProcess.exec(command, (error, stdout, stderr) => {
+    if (error !== null) {
+      cb(stderr)
+    }
+    cb(null, stdout)
+  })
+}
+
+function timeout() {
+  return new Promise((resolve, reject) => setTimeout(resolve, 100))
```

Comment:
I understand the idea behind creating a promise to block and wait for 100 milliseconds but, would not be better to call all the logic in setTimeout callback to avoid having block code?